Core/FeatureFlag.swift

```diff
@@ -49,6 +49,10 @@ extension FeatureFlag: FeatureFlagSourceProviding {
 
 extension FeatureFlagger {
     public func isFeatureOn(_ featureFlag: FeatureFlag) -> Bool {
-        isFeatureOn(forProvider: featureFlag)
+        if isDebugBuild {
+            return true
+        }
```

Comment:
@graeme I keep having to turn on internal features manually, due to swapping devices, cleaning the build etc. Do you see a problem with forcing all features to on when running in debug mode?

Reply:
Gonna take this out before merging to make sure it's cool by others on the team first.